src/target/riscv/riscv.c

```diff
@@ -1576,6 +1629,8 @@ static int riscv_trigger_detect_hit_bits(struct target *target, int64_t *unique_
 
 	// FIXME: Add hit bits support detection and caching
```

Comment:
This TODO can now be removed:

- Hit bit detection you have just implemented.
- Caching cannot be implemented, as we have [already discussed](https://github.com/riscv-collab/riscv-openocd/pull/1144#discussion_r1832259614).

```suggestion
```

Reply:
addressed

Reply:
@JanMatCodasip, are you sure? As outlined [here](https://github.com/riscv/riscv-debug-spec/issues/1080#issuecomment-2462506319) in your spec proposal

Reply:
Created an issue instead of this `TODO`: https://github.com/riscv-collab/riscv-openocd/issues/1169

Reply:
@en-sc, perhaps I initially misunderstood your comment. 

Did you mean it this way? For a given triplet of `tselect`, `tdata1` and `tdata2`, the support of `hit` bit would be detected at the time the trigger is set using the procedure described in https://github.com/riscv/riscv-debug-spec/issues/1080#issuecomment-2462506319, and then:

- remembered for the whole time the trigger remains set,
- and perhaps also for future, in case that same trigger (defined by `tselect`, `tdata1` and `tdata2`) becomes set again.

Reply:
Yes, that's the idea. You've put it much more clearly. Thanks!